core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -73,4 +84,38 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
         error: (_: unknown) => (row.bookmarked = wasPreviouslyBookmarked),
       });
   }
+
+  /* delete a single execution and display current workflow execution */
+
+  onDelete(row: WorkflowExecutionsEntry) {
+    if (this.workflow.wid === undefined) {
+      return;
+    }
+    this.workflowExecutionsService
+      .deleteWorkflowExecutions(this.workflow.wid, row.eId)
+      .pipe(untilDestroyed(this))
+      .subscribe({
+        complete: () => this.workflowExecutionsList?.splice(this.workflowExecutionsList.indexOf(row), 1),
+      });
+  }
+
+  confirmUpdateWorkflowExecutionsCustomName(row: WorkflowExecutionsEntry, name: string, index: number): void {
+    if (this.workflow.wid === undefined) {
+      return;
+    }
+    this.workflowExecutionsService
+      .updateWorkflowExecutionsName(this.workflow.wid, row.eId, name || this.defaultWorkflowExecutionsName)
```

Comment:
why pass name or untitled? maybe you can explain the use case?

Reply:
discussed

Reply:
seems passing only name is enough